modules/social_features/social_user_export/src/ExportUser.php

```diff
@@ -21,6 +21,8 @@ class ExportUser {
    *   UserInterface entity.
    * @param array $context
    *   Context of the operation.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
```

Comment:
Hey Brhane,

Can you explain which line can throw an InvalidPluginDefinitionException?

~ Alexander

Reply:
Hey Alexander,
line 102, at the snippet: \Drupal::entityTypeManager()->getStorage('profile');

Reply:
So this would mean that if the profile module is not installed on the platform then the user export can not be used? Can you find a solution to that? ;-)

Reply:
Hey, Alexander,
I have tried to fix what you mentioned by specifying that profile module is required by user export module.
Brhane

Reply:
Hey Brhane,

I actually liked the fact that we didn't have a dependency on the Social Profile module. Perhaps we can do something like a `try catch` statement ( http://php.net/manual/en/language.exceptions.php ). This way we can use the profile field if it exists and if we don't have it enabled we just return the empty string value :)

Make sure you're specific enough in which error you catch so we don't mask other errors to other developers.